src/new-client/src/Web/TearOutSection/state.ts

```diff
@@ -0,0 +1,31 @@
+import { bind } from "@react-rxjs/core"
+import { createSignal } from "@react-rxjs/utils"
+import { map, filter } from "rxjs/operators"
+
+type TearOutSectionEntry = [boolean, Section]
+
+export type Section = "liverates" | "trades" | "analytics"
+
+export const [tearOutSectionEntry$, tearOutSection] = createSignal(
+  (bool: boolean, section: Section): TearOutSectionEntry => [bool, section],
+)
+
+export const [useTearOutSectionEntry] = bind<TearOutSectionEntry | null>(
+  tearOutSectionEntry$,
+  null,
+)
+
+export const sectionConfig = {
```

Comment:
Explicit type here too

Reply:
Done, defined these types and certain consts have explicitly these
```
type tornOutSize = { width: number, height: number}

export type Section = "liverates" | "trades" | "analytics"

export type SectionConfig = { [section in Section]: tornOutSize }

export type TornOutSection = { [section in Section]: boolean }